test/fp64-test/fp64-shader.spec.js

```diff
@@ -246,9 +243,10 @@ function test_float_add(gl, testName) {
   const float0_vec2 = fp64ify(float0);
   const float1_vec2 = fp64ify(float1);
   const float_ref_vec2 = fp64ify(float_ref);
+
   const program = new Program(gl, assembleShaders(gl, {
-        vs: readFileSync(join(__dirname, './shader-tests/vs_float_add.glsl'), 'utf8'),
-        fs: readFileSync(join(__dirname, './shader-tests/fs.glsl'), 'utf8'),
+        vs: require('./vs_float_add.glsl'),
```

Comment:
This won't work in Node.js without bundling which is a pity (and probably won't play well with browserify which is a smaller concern at this point).

Unfortunately, it seems there is no perfect option to inline text files across both node and browser, so this is not a bad way.